Jint.Tests/Runtime/EngineLimitTests.cs

```diff
@@ -9,21 +9,12 @@ public class EngineLimitTests
     [Fact]
     public void ShouldAllowReasonableCallStackDepth()
     {
-        if (OperatingSystem.IsMacOS())
```

Comment:
I think for this test case we might want to disable the stack guard (value -1?) to ensure we don't start writing silly code - this is also a performance test when it comes to stack handling. So we would keep this test as-is expect for configuring engine with disabled stack guard.

Can of course give the script generation improvements (x starts with 1 and result checked).

Reply:
Disable options (`-1`) make sense. I add disable option and also kept the actual test case as-is.